client/crowi-form.ts

```diff
@@ -212,17 +212,18 @@ $(function() {
     } else {
       if (currentLine && currentLine.text.charAt(0) === '|') {
         // next cell in table
-        var newPos = text.indexOf('|', pos.start + 1)
+        const newPos = text.indexOf('|', pos.start + 1)
         if (newPos < 0 || newPos === text.lastIndexOf('|', currentLine.end - 1)) {
           $target.selection('setPos', { start: currentLine.end, end: currentLine.end })
         } else {
           $target.selection('setPos', { start: newPos + 2, end: newPos + 2 })
         }
       } else {
         // indent
+        const isBlankLine = !currentLine?.text
         $target.selection('replace', {
           text:
-            '    ' +
+            `${isBlankLine ? '\n' : ''}    ` +
```

Comment:
nits:
```suggestion
            isBlankLine ? '\n' : '    ' +
```
I think this is easier to understand.